src/java.base/share/classes/java/lang/invoke/InnerClassLambdaMetafactory.java

```diff
@@ -92,6 +93,11 @@
     private static final ProxyClassesDumper dumper;
 
     private static final boolean disableEagerInitialization;
+    private static final boolean stableLambdaName;
```

Comment:
What do you think about `boolean generateStableLambdaNames`?

Reply:
It's more precise, renamed.